### ISSUE TITLE
Selectonemenu: filter="true" causes dropdown to collapse immediately

### ISSUE THREAD

**Issue by elias-jk** [primefaces/primefaces#7835]
**Describe the defect**
When creating a SelectOneMenu that allows filtering via a textfield (property `filter="true"`), the menu immediately collapses on mobile devices after opening.

**Environment:**
 - PF Version: _10.0_
 - Server: _Payara 5.194_
 - Affected browsers: _e.g. Chrome on Android (and Desktop in mobile mode), Firefox on Android_
 - Unaffected browsers: _e.g. Chrome on Desktop, Firefox on Desktop (and in mobile mode)_

**To Reproduce**
1. Have a filterable `p:selectOneMenu`
2. Try to expand it by tapping on it

**Expected behavior**
The menu expands. Searching and selecting should be possible as it is in desktop browsers.

**Actual behaviour (Chrome on Android /)**
The menu as well as the keyboard collapses.

**Actual behaviour (Chrome)**
The menu collapses, but the keyboard remains open.

**Example XHTML**
```html
<p:selectOneMenu id="parent" value="#{categoryBean.parent}" style="width: 100%;"
                 filter="true" filterMatchMode="contains">
    <f:selectItems value="#{categoryBean.parents}" var="parent" itemLabel="#{parent}" itemValue="#{parent}" />
</p:selectOneMenu>
```

**Example Bean**
```java
@Named
@ViewScoped
public class CategoryBean implements Serializable {

    @Getter
    @Setter
    private List<String> parents = List.of("One", "Two", "Three");

    @Getter
    @Setter
    private String parent;
}
```

**Comment by christophs78** [comment]
Reproduceable with http://primefaces.org/showcase/ui/input/oneMenu.xhtml ("Advanced") on Android/Chrome

**Comment by melloware** [comment]
isn't this fixed already: https://github.com/primefaces/primefaces/issues/7298

**Comment by christophs78** [comment]
Yes. Fixed for 11.0.0.

![SelectOneMenuFilter_01](https://user-images.githubusercontent.com/10461942/133640552-8da70805-86aa-4ec6-b8cc-fca78d83a092.png)
![SelectOneMenuFilter_02](https://user-images.githubusercontent.com/10461942/133640563-3e83a94d-1926-4d00-9d57-933f32196ab0.png)
